flink-connectors/flink-connector-rabbitmq/pom.xml

```diff
@@ -92,4 +92,31 @@ under the License.
 
 	</dependencies>
 
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>shade-flink</id>
+						<configuration>
+							<artifactSet>
+								<includes combine.children="append">
+									<include>com.rabbitmq:amqp-client</include>
+								</includes>
+							</artifactSet>
+							<relocations>
+								<relocation>
+									<pattern>com.rabbitmq</pattern>
+									<shadedPattern>org.apache.flink.rabbitmq.shaded.com.rabbitmq</shadedPattern>
+								</relocation>
+							</relocations>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+
```

Comment:
NOTE: this change was added so that class loading would work properly in the added unit tests (related article https://www.codetd.com/en/article/7358211)

Reply:
This makes the RabbitMQ connector becoming a fat jar. We need to do the following for a fat jar:
- Usually it will create a fat jar in a separate module, e.g flink-sql-connector-kinesis vs flink-connector-kinesis
- Need to add a NOTICE file which declares the bundled libraries and their licenses

Reply:
@dianfu Ah, I see :+1: I addressed your comments in the latest change